### DISCUSSION TITLE
What index type is optimal for face detection? (for milvus standalone)

### DISCUSSION THREAD

**Discussion by martinenkoEduard** [milvus-io/milvus#18387] [Q&A and General discussion]
What index type is optimal for face detection?
collection with less than 5 millions of entries works fine with Flat index.

What index can you suggest?

**Comment by yhmo** [comment]
No best index, just choose a kind of index that suitable to you.
The precision: IVF_FLAT/HNSW > IVF_SQ8 > IVF_PQ
The memory usage: IVF_FLAT/HNSW > IVF_SQ8/IVF_PQ
The performance: IVF_PQ/HNSW > IVF_FLAT/IVF_SQ8
